package.json

```diff
@@ -27,7 +27,8 @@
     "format:md": "yarn lint:md --fix",
     "proto-gen": "yarn exec ./scripts/protocgen.sh",
     "prettier": "prettier --write . --ignore-path .gitignore",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "semantic-release": "lerna exec --concurrency 1 -- semantic-release -e semantic-release-monorepo --tag-format='${LERNA_PACKAGE_NAME}-v\\${version}'"
```

Comment:
Why are we limiting concurrency?

Reply:
No specific reason, just followed their documentation https://github.com/pmowrer/semantic-release-monorepo#tagformat

Reply:
don't think it will cause an issue as I tested with this concurrency limit